config/tech-docs.yml

```diff
@@ -0,0 +1,37 @@
+# Host to use for canonical URL generation (without trailing slash)
+host: docs.developer.service.hmrc.gov.uk
+
+# Header-related options
+show_govuk_logo: true
+service_name: HMRC Developer Hub
+service_link: https://developer.service.hmrc.gov.uk/api-documentation
+
+# Links to show on right-hand-side of header
+header_links:
+  VAT (MTD) End-to-End Service Guide: /guides/vat-mtd-end-to-end-service-guide
```

Comment:
Is this here intentionally?  If so, I wouldn't have thought we would be wanting to link between service guides because that is quite a nasty maintenance overhead.